src/components/navigation/NotificationButton.tsx

```diff
@@ -41,9 +89,29 @@ const StyledNotificationButton = styled.div`
 export function NotificationButton() {
   const components = useBosComponents();
 
+  const nHref = components.nearOrg.notifications.page;
+
   return (
     <StyledNotificationButton className="nav-notification-button">
-      <VmComponent src={components.notificationButton} />
+      <Button href={nHref} aria-label="View Notifications">
+        <svg viewBox="0 0 25 24" fill="none">
+          <path
+            d="M5.60205 9.74986C5.6008 8.86072 5.77547 7.98011 6.11601 7.15876C6.45654 6.33741 6.95622 5.59155 7.58624 4.96414C8.21627 4.33673 8.9642 3.84016 9.78696 3.50304C10.6097 3.16592 11.491 2.99491 12.3802 2.99986C16.0927 3.02799 19.0645 6.11236 19.0645 9.83424V10.4999C19.0645 13.8561 19.7677 15.8061 20.3864 16.8749C20.4521 16.9887 20.4868 17.1178 20.4869 17.2492C20.487 17.3806 20.4526 17.5098 20.3871 17.6237C20.3216 17.7377 20.2273 17.8324 20.1136 17.8984C20 17.9644 19.871 17.9994 19.7395 17.9999H4.92705C4.79562 17.9994 4.66662 17.9644 4.55297 17.8984C4.43933 17.8324 4.34502 17.7377 4.27951 17.6237C4.21399 17.5098 4.17957 17.3806 4.17969 17.2492C4.1798 17.1178 4.21446 16.9887 4.28017 16.8749C4.89892 15.8061 5.60205 13.8561 5.60205 10.4999V9.74986Z"
+            stroke="currentColor"
+            strokeWidth="2"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          />
+          <path
+            d="M9.33331 18V18.75C9.33331 19.5456 9.64938 20.3087 10.212 20.8713C10.7746 21.4339 11.5377 21.75 12.3333 21.75C13.129 21.75 13.892 21.4339 14.4546 20.8713C15.0172 20.3087 15.3333 19.5456 15.3333 18.75V18"
+            stroke="currentColor"
+            strokeWidth="2"
+            strokeLinecap="round"
+            strokeLinejoin="round"
+          />
+        </svg>
+      </Button>
+      {/* <VmComponent src={components.notificationButton} /> */}
```

Comment:
@marcinbodnar Here is a new notification button with preview https://github.com/near/near-discovery-components/pull/323

Reply:
@shelegdmitriy It's not ready yet, I didn't add reviewers